specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/ipTags.json

```diff
@@ -0,0 +1,406 @@
+            "description": "Request successful. The operation returns the resulting IpTags resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get_IpTag": {
+            "$ref": "./examples/IpTagsGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_CreateOrUpdate",
+        "description": "Creates or updates an ipTag in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ipTagName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ipTag."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            },
+            "description": "Parameters supplied to the create or update IpTags operation."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns the resulting ipTag resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting ipTag resource.",
+            "schema": {
+              "$ref": "#/definitions/IpTags"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "CreateOrUpdate_IpTag": {
+            "$ref": "./examples/IpTagsCreate.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "IpTags"
+        ],
+        "operationId": "IpTags_Update",
+        "description": "Updates an ipTag in a specified resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "ipTagName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the ipTag."
+          },
+          {
+            "name": "ipTagsParameters",
```

Comment:
For the sake of consistency accross specs, please name this as `parameters`

Reply:
Done